base/show.jl

```diff
@@ -490,7 +490,7 @@ function show_unquoted(io::IO, ex::Expr, indent::Int, prec::Int)
         show_call(io, head, args[1], args[2:end], indent)
```

Comment:
It doesn't look like this clause is necessary anymore, since (I think) `calldecl` should be handled properly below.